packages/flutter_test/lib/src/_goldens_io.dart

```diff
@@ -253,3 +277,24 @@ class DefaultWebGoldenComparator extends WebGoldenComparator {
     throw UnsupportedError('DefaultWebGoldenComparator is only supported on the web.');
   }
 }
+
+/// Reads the red value out of a 32 bit rgba pixel.
+int readRed(int pixel) => pixel >> 24 & 0xff;
+
+/// Reads the green value out of a 32 bit rgba pixel.
+int readGreen(int pixel) => pixel >> 16 & 0xff;
+
+/// Reads the blue value out of a 32 bit rgba pixel.
+int readBlue(int pixel) => pixel >> 8 & 0xff;
+
+/// Reads the alpha value out of a 32 bit rgba pixel.
+int readAlpha(int pixel) => pixel & 0xff;
+
+/// Convenience wrapper around [decodeImageFromPixels].
+Future<Image> _createImage(ByteData bytes, int width, int height) {
```

Comment:
This does not appear to be creating the right images, or the invert function may not be getting the right pixel.

Reply:
Whoops! Definitely a bug, it should be fixed now though.